src/controllers/media.ts

```diff
@@ -239,7 +239,9 @@ export const getBySanitizedTitle = async(ctx: Context): Promise<MediaMetadataInt
 
   try {
     imdbData.searchMatches = [title];
-    const newlyCreatedResult = await MediaMetadata.create(imdbData);
+    let newlyCreatedResult = await MediaMetadata.create(imdbData);
```

Comment:
Can you leave a comment around here with a link to the mongoose issue so we are reminded of why this isn't more elegant?

Reply:
Done & https://github.com/Automattic/mongoose/issues/9118